src/components/settings/Calendars.js

```diff
@@ -203,14 +219,61 @@ export default class Calendars extends AddDeleteSetting {
     this.setState({ hexColor })
   }
 
+  handleFieInputChange = event => {
+    const { verifyNewCalendar } = this.props
+
+    console.log('selected files', event.target.files[0])
+    if (event.target.files[0]) {
+      let file = event.target.files[0]
+      let fileExt = file.name.split('.').pop()
+
+      if (fileExt.toLowerCase() !== 'ics') {
```

Comment:
Maybe just add a warning. We should check the content of the file, not the name. Do not return early here.